test/parse.js

```diff
@@ -367,6 +367,7 @@ test('parse()', function (t) {
 
     t.test('allows for query string prefix', function (st) {
         st.deepEqual(qs.parse('?foo=bar', { ignoreQueryPrefix: true }), { foo: 'bar' });
+        st.deepEqual(qs.parse('???foo=bar', { ignoreQueryPrefix: true }), { foo: 'bar' });
```

Comment:
This doesn't seem like the right test. At most, it should *throw* if it encounters > 0 question marks, or > 1 when ignoreQueryPrefix (but that'd be a semver-major change).

I'm pretty sure question marks after the first one are a valid part of the query string, that the server can receive.

Reply:
Ok. Thanks for review :)